src/sdk/pynni/nni/networkmorphism_tuner/layers.py

```diff
@@ -608,6 +603,7 @@ def is_layer(layer, layer_type):
         return isinstance(layer, (StubFlatten,))
     elif layer_type == "GlobalAveragePooling":
         return isinstance(layer, StubGlobalPooling)
+    return None  # FIXME: return or raise?
```

Comment:
And this

Reply:
return None. do not change the logic, thank you.